compass/ocean/mesh/lts_regions.py

```diff
@@ -0,0 +1,310 @@
+import math
+import subprocess as sp
+
+import netCDF4 as nc
+import numpy as np
+import xarray
+from shapely.geometry import Point
+from shapely.geometry.polygon import Polygon
+
+from compass.step import Step
+
+
+class LTSRegionsStep(Step):
+    """
+    A step for adding LTS regions to a global MPAS-Ocean mesh
+
+    Attributes
+    ----------
+    cull_mesh_step : CullMeshStep
+        The culled mesh step containing input files to this step
+    """
+    def __init__(self, test_case, cull_mesh_step,
+                 name='lts_regions', subdir='cull_mesh'):
+        """
+        Create a new step
+
+        Parameters
+        ----------
+        test_case : compass.ocean.tests.global_ocean.mesh.Mesh
+            The test case this step belongs to
+
+        cull_mesh_step : cull_mesh_step
+            The culled mesh step containing input files to this step
+
+        name : str, optional
+            the name of the step
+
+        subdir : str, optional
+            the subdirectory for the step.  The default is ``name``
+        """
+        super().__init__(test_case, name=name, subdir=subdir,
+                         cpus_per_task=None, min_cpus_per_task=None)
```

Comment:
For this step, it makes sense to default to 1 task and 1 cpu per task.
```suggestion
        super().__init__(test_case, name=name, subdir=subdir)
```

Reply:
Done.